docs/how-we-work.md

```diff
@@ -6,31 +6,39 @@ Our basic principles are those familiar to anybody who has contributed to a prom
 
 -   We produce open source software, created and maintained in repositories where it may be inspected by the public who places their trust in it and copied for use by other agencies or institutions.
 -   We adhere to the basic practices of agile software development, using the Scrum development framework.
--   We practice human-centered design. Everything that we produce is highly accessible, per [WCAG 2.1](https://www.w3.org/TR/WCAG21/).
+-   We practice human-centered design. Everything that we produce is highly accessible, per [WCAG 2.0](https://www.w3.org/TR/WCAG20/).
```

Comment:
WCAG 2.1 has a number of improvements over 2.0 and is backwards-compatible. If we're intentionally not adopting it we should document why.

https://www.w3.org/WAI/standards-guidelines/wcag/#versions
> Content that conforms to WCAG 2.1 also conforms to WCAG 2.0. (This is often called “backwards compatible”.) A website that meets WCAG 2.1 should meet the requirements of policies that reference WCAG 2.0.
>
> To put it another way: If you want to meet both WCAG 2.0 and WCAG 2.1, you can use the 2.1 resources and you don’t need to bother looking at 2.0.
> 
> WCAG 2.0 and WCAG 2.1 are both existing standards. WCAG 2.1 does not deprecate or supersede WCAG 2.0. W3C encourages you to use the most recent version of WCAG when developing or updating content or accessibility policies.

Reply:
Something to catch about with Zuhair when we review our own accessibility policy. When we talked, the library officially meets 2.0 and has yet to make the policy change to meet 2.1.

Reply:
Since 2.1 is a superset of 2.0 I would look forward and let the main policy catch up.